packages/vscode-extension/src/webview/components/UrlSelectItem.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { PropsWithChildren } from "react";
+import { UrlSelectFocusable } from "./UrlSelect";
+import { NavigationHistoryItem } from "../../common/Project";
+
+interface UrlSelectItemProps {
+  item: NavigationHistoryItem;
+  width: number;
+  style?: React.CSSProperties;
+  itemList: HTMLDivElement[];
+  refIndex: number;
+  textfieldRef: React.RefObject<HTMLInputElement>;
+  onConfirm: (item: NavigationHistoryItem) => void;
+  onArrowPress: (
+    e: React.KeyboardEvent,
+    prev?: UrlSelectFocusable,
+    next?: UrlSelectFocusable
+  ) => void;
+  getNameFromId: (id: string) => string;
+  noHighlight?: boolean;
+}
+
+const UrlSelectItem = React.forwardRef<HTMLDivElement, PropsWithChildren<UrlSelectItemProps>>(
```

Comment:
```suggestion
const UrlSelectItem = 
```
This isn't `forwardRef` anymore, right?